.github/workflows/dest_postgresql.yml

```
@@ -45,7 +45,8 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           version: v1.49.0
-          args: "--config ${GITHUB_WORKSPACE}/plugins/.golangci.yml"
+          working-directory: plugins/destination/postgresql
+          args: "--config ../.golangci.yml"
```

```suggestion
          args: "--config ../../.golangci.yml"
```

It should be two levels up